### PR TITLE
Cleans up result folder handling

### PR DESCRIPTION
**Pull request by vulder**

### CODE REVIEW

File: varats-core/varats/experiment/experiment_util.py [hunk 1]
```diff
@@ -26,12 +26,34 @@
 from varats.utils.settings import vara_cfg, bb_cfg
 
 
+def get_vara_result_folder(project: Project) -> Path:
```
Comment (boehmseb) [inline varats-core/varats/experiment/experiment_util.py]: A test for this function would be nice. This should be easy enough to do.

Comment (boehmseb) [reply varats-core/varats/experiment/experiment_util.py]: This is maybe a bit picky, but is it good to include `vara` in the function name? Other tools/experiments also put their results there.

**Review by boehmseb — CHANGES_REQUESTED** [review]

**Review by cormensratio — APPROVED** [review]
Lgtm

File: varats-core/varats/experiment/experiment_util.py (hunk 1 above)
Comment (vulder) [reply varats-core/varats/experiment/experiment_util.py]: done

Comment (vulder) [reply varats-core/varats/experiment/experiment_util.py]: varats is probably better, that's what I aimed for anyhow

**Review by boehmseb — APPROVED** [review]

**Pull request merged**